pkg/kudoctl/cmd/install/install.go

```diff
@@ -20,7 +19,7 @@ type Options struct {
 	AutoApprove     bool
 	InstanceName    string
 	Namespace       string
-	Parameters      []string
+	Parameters      map[string]string
```

Comment:
I played with an idea to make it a proper `type Parameter struct` but struggled to find meaningful fields other than key and value. So it is a map for now.